resources/js/visual-editor/editor/post-title.css

```
@@ -0,0 +1,73 @@
+    color: var(--ap-visual-editor-post-title-foreground);
+    border: 0;
+    border-radius: var(--rounded-btn, 6px);
+    outline: none;
+    font-family: inherit;
+    font-size: clamp(28px, 4vw, 40px);
+    font-weight: 700;
+    line-height: 1.2;
+    letter-spacing: -0.01em;
+    resize: none;
+    overflow: hidden;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)